src/pages/type-classes/cats.md

```diff
@@ -35,13 +34,9 @@ The companion object of every Cats type class has an `apply` method
 that locates an instance for any type we specify:
 
 ```scala mdoc
-val showInt = Show.apply[Int]
+given showInt: Show[Int] = Show.apply[Int]
 ```
 
-Oops---that didn't work!
-The `apply` method uses *implicits* to look up individual instances,
-so we'll have to bring some instances into scope.
-
```

Comment:
Need to revise this,

if left as `val showInt` then `Show.apply` is left as a lamdba (unevaluated) and the output of mdoc didn't match the text.

If changed to `given showInt` then `Show.apply` uses showInt as a cyclic reference and succeeds (but warns on compilation about an infinite loop)